City-of-Helsinki/drupal-helfi-strategia
# UHF-3396: Blacklist /sisallonhallinta path from search engines

To test this:

- `composer install`
- `drush cim`
- Create few pages with path starting with `/sisallonhallinta`. For example `/sisallonhallinta`, `/sisallonhallinta/test`, `/sisallonhallinta/second/third`.

Make sure response headers include `X-Robots-Tag: noindex, nofollow` header:

![2021-11-24-10-33-13-screenshot](https://user-images.githubusercontent.com/771113/143202718-e0ef39c7-d54c-4e9f-8d2b-da6d50709c47.png)
